src/utils/gaa.js

```diff
@@ -231,10 +247,9 @@ export class GaaMeteringRegwall {
   /**
    * Configures Google Sign-In.
    * @nocollapse
-   * @param {{ redirectUri: string }=} params
    * @return {!Promise}
    */
-  static configureGoogleSignIn({redirectUri} = {redirectUri: ''}) {
+  static configureGoogleSignIn() {
```

Comment:
Should this be made private? I see that it is getting called within renderGoogleSignInButton_(). Should users be able to call this function or only allow show()?

Reply:
I just privatized it